morse_decode.rb

```diff
@@ -0,0 +1,29 @@
+MORSE_CODES = {
+  '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E',
+  '..-.' => 'F', '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J',
+  '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N', '---' => 'O',
+  '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T',
+  '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y',
+  '--..' => 'Z',
+  ' ' => ' '
+}.freeze
+
+# Decoding letter
+def decode_string(morse_string)
+  MORSE_CODES[morse_string] || ''
+end
+
+morse_code1 = '.-'
+decoded_string = decode_string(morse_code1)
+puts decoded_string
+
+# Decoding word
+def decode_word(morse_word)
+  morse_word.split.map { |morse_string| decode_string(morse_string) }.join
+end
+
+morse_code2 = '-- -.--'
+decoded_word = decode_word(morse_code2)
+puts decoded_word
+
```

Comment:
- You have done great so far :+1: :+1:  I appreciate the effort you've put into implementing the Morse code decoding functionality. but there are still a few requirements, you need to meet before moving to the next milestone. :+1: :+1: 

- - 
![Selection_030](https://github.com/Pi1998/decode-morse/assets/107632780/302cf9d4-f204-4b28-8aa3-d3d3a1d84746)

- [ ] I noticed that you are currently missing one of the required methods based on the project specifications. As per the project requirements, you need three distinct methods for decoding:
1. decode_char: To decode a Morse code character.
2. decode_word: To decode an entire word in Morse code.
3. decode_message: To decode the entire message in Morse code.

Currently, you have implementations for `decode_char` and `decode_word`, but you are missing the decode_message method. I'll suggest you implement this. :+1: :+1: :+1: :smile: 

- [ ] Also looking at the screenshot below, you can see that after decoding a sentence, the words are parked together without the correct spacing, and this is not correct. As required, after decoding a sentence, it should be properly spaced and readable. I'll suggest you look into this too. :smile: :wave: 

- - Screenshot
![Selection_028](https://github.com/Pi1998/decode-morse/assets/107632780/1e1d879b-df6e-48dc-8120-7055f6169e43)